process/block/displayBlock.go

```diff
@@ -51,32 +63,34 @@ func DisplayLogInfo(
 	numShards uint32,
 	selfId uint32,
 	dataPool dataRetriever.PoolsHolder,
+	txCounter *transactionCounter,
```

Comment:
this func could have been made part of the transactionCounter structure. And all methods could have been made unexported.

Reply:
Changed